bucoffea/plot/studies/stack_plot/get_hf_noise_estimate.py

```diff
@@ -178,35 +155,33 @@ def get_hf_noise_estimate(acc, outtag, outrootfile, distribution, years=[2017, 2
 
         outrootfile[f'hf_estimate_{distribution}_{year}'] = (sumw, xedges)
 
+def commandline():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('inpath', help='Path to the merged input accumulator.')
+    parser.add_argument('--years', nargs='*', type=int, default=[2017,2018], help='Years to run.')
+    parser.add_argument('--region', default='cr_vbf_qcd', help='Name of the HF-noise enriched control region as defined in the VBF H(inv) processor.')
+    parser.add_argument('--distribution', default='.*', help='Regex specifying the list of distributions to run.')
+    args = parser.parse_args()
+    return args
+
 def main():
-    args = parse_cli()
+    args = commandline()
     inpath = args.inpath
     acc = dir_archive(inpath)
     acc.load('sumw')
     acc.load('sumw2')
 
-    outtag = re.findall('merged_.*', inpath)[0].replace('/','')
-    outdir = f'./output/{outtag}/hf_estimate'
-    if not os.path.exists(outdir):
-        os.makedirs(outdir)
+    outdir = pjoin('./output/',args.inpath.replace('..','').replace('/',''),'hf_estimate')
+    dump_info(args, outdir)
 
     outrootpath = pjoin(outdir, 'vbfhinv_hf_estimate.root')
```

Comment:
Quick question: If I understand this correctly, you're saving the file under `output/{outtag}/hf_estimate`, and in the above script (where `plot_data_mc` function is defined) I think you're reading the file from `{outtag}/hf_estimate`. Did I get that right? Just wanted to check if it's working :)

Reply:
the folder should be the same in the end. maybe the naming convention could do a bit of polishing